acceptance/tests/partitions/partitions_connect_test.go

```diff
@@ -91,6 +91,11 @@ func TestPartitions_Connect(t *testing.T) {
 				"global.enableConsulNamespaces":  "true",
 				"global.logLevel":                "debug",
 
+				// Currently there is a bug with xds limits being reached for a single server, so need to increase
+				// the server replicas to 3
+				// TODO: remove when NET-5819 is fixed
+				"global.replicas": "3",
```

Comment:
**question:** Should this be for CNI only or does it make all the partition tests better?

Reply:
it could be for CNI only, I don't "think" the test was flaking on tests other than CNI... however this doesn't hurt anything so thought it would be simpler to not have a separate check for CNI being enabled.

Reply:
It's worth noting that this will make local test runs fail unless people are spinning up their kind clusters with 3+ nodes, but I don't see it being a huge ordeal.